### ISSUE TITLE
Docker compose file needed

### ISSUE THREAD

**Issue by HakaishinShwet** [nus-apr/auto-code-rover#10]
For easy and better selfhosting docker compose is needed for this awesome project ;-))

**Comment by arsaboo** [comment]
+1000

**Comment by jimwhite** [comment]
By using docker compose do you mean so that the tasks are run in their own containers?  I agree but it seems to me that is a SWE-bench issue because that is where the environments are created and managed.  Right now it is all Python based (using Conda) but generalizing to Docker containers will make this more useful.

I've just made a PR for an improved Dockerfile for this project so it will run on other architectures (i.e. arm64, ppc):
https://github.com/nus-apr/auto-code-rover/pull/22

**Comment by HakaishinShwet** [comment]
@jimwhite yeah that's what i mean . Docker compose file simplifies things for everyone in deployment so just thought if it will be possible then it will be great
